build.rs

```diff
@@ -8,13 +10,36 @@ enum DalekBits {
     Dalek64,
 }
 
+// Only when the curve25519_dalek_bits is not explicitly set.
+// Error handling when the bits setting cannot be determined
 #[cfg(all(not(curve25519_dalek_bits = "64"), not(curve25519_dalek_bits = "32")))]
-#[deny(dead_code)]
-fn lotto_curve25519_dalek_bits() -> DalekBits {
+fn determine_curve25519_dalek_bits_error(error_code: i32, trigger_msg: &str) -> ! {
+    eprintln!("Error: {trigger_msg}");
+    eprintln!("Please set cfg(curve25519_dalek_bits) explicitly either as 32 or 64.");
+    std::process::exit(error_code)
+}
+
+// Only when the curve25519_dalek_bits is not explicitly set.
+#[cfg(all(not(curve25519_dalek_bits = "64"), not(curve25519_dalek_bits = "32")))]
+fn determine_curve25519_dalek_bits() -> DalekBits {
     use platforms::target::PointerWidth;
 
-    let target_triplet = std::env::var("TARGET").unwrap();
-    let platform = platforms::Platform::find(&target_triplet).unwrap();
+    // TARGET environment is supplied by Cargo
+    // https://doc.rust-lang.org/cargo/reference/environment-variables.html
+    let target_triplet = match std::env::var("TARGET") {
+        Ok(t) => t,
+        Err(_) => determine_curve25519_dalek_bits_error(
+            10,
```

Comment:
What's the purpose of the constants exactly? If it makes sense, I'd rather have them be `const`s up top